site/css/style.sass

```diff
@@ -81,9 +81,9 @@ footer.footer
 .toc
   overflow-y: scroll
   bottom: 0
-  top: $navbar-height + 5rem
+  top: 50px
```

Comment:
Change the $navbar-height variable instead

Reply:
Done.